mjpc/test/agent/norm_test.cc

```diff
@@ -94,6 +94,7 @@ TEST_P(NormTest, Hessian) {
   }
 }
 
+#ifndef _MSC_VER
```

Comment:
Could you write a comment as to why this is needed.

Reply:
I was getting the same issue @saran-t was getting here: https://github.com/deepmind/mujoco_mpc/pull/21

It was inscrutable to me, but I bisected commenting out regions of the file until I came to `INSTANTIATE_TEST_SUITE_P`, which is apparently only ever used in `NormTest.cc`.

I guess the Google Testing System has trouble with MSVC?   Perhaps it's fixed in an updated version. 🤷‍♂️